minwoo.lee_java/src/BOJ2056.java

```diff
@@ -0,0 +1,61 @@
+import java.io.*;
+import java.util.*;
+
+public class BOJ2056 {
+    static int N;
+    static int[] inDegree, jobTime, endTime;
+    static ArrayList<Integer>[] nextJob;
+
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        N = Integer.parseInt(br.readLine());
+        init();
+        StringTokenizer st;
+        for (int i = 1; i <= N; i++) {
+            st = new StringTokenizer(br.readLine());
+            int time = Integer.parseInt(st.nextToken());
+            jobTime[i] = time;
```

Comment:
```suggestion
            jobTime[i] = Integer.parseInt(st.nextToken());
```
이렇게 바로 써주었어도 됐을 것 같습니다!